specs/FamilySafety.md

```diff
@@ -0,0 +1,82 @@
+Family Safety
+===
+
+# Background
+Provide end evelolper a new API to toggle Family Safety feature on and off. Once Family Safety is enabled, developer won't be able to turn it off while webview2 instance is running. Once enable, it will provide the same functionally as the browser like: Activity report, Safe Search and Web Filtering. Besides toggling the feature, we also provide an API to add desired sites to the soft bypass list, so app functionality will not be impacted by Family Safety restricted mode. Parents can still over the site if they prefer.
+Please see https://www.microsoft.com/en-us/microsoft-365/family-safety for details on Family Safety. 
+
+# Examples
+## WinRT and .NET   
+```c#
+void CreateEnvrionmentWithOption()
+{
+    CoreWebView2EnvironmentOptions options = new CoreWebView2EnvironmentOptions();
+    options.IsFamilySafetyEnabled = true;
+    options.SetFamilySafetySoftByPassList(1, "appassets.example/AppStartPage.html");
```

Comment:
This is a good example of use of the bypass list. Can you add a comment making it clear where this URL comes from?
```suggestion
    // Later we use SetVirtualHostNameToFolderMapping with `appassets.example` to map to local app content.
    // We want the Family Safety filter on generally, but not for app content.
```

Reply:
I added some comment. Let me know what you think. I left out the SetVirtualHostNameToFolderMapping  part since I don't think its relevant to this API